bundle/EventListener/Shop/ProductIndexListener.php

```diff
@@ -54,6 +54,7 @@ public function onProductIndex(ResourceControllerEvent $event): void
         }
 
         $currentRequest->attributes->set('nglayouts_sylius_taxon', $taxon);
+        $currentRequest->attributes->set('nglayouts_sylius_resource', $taxon);
```

Comment:
If we're already doing this in `ResourceShowListener`, why do we have to do it here too?